README.rst

```diff
@@ -126,14 +123,13 @@ More
 Functions are well documented, see docstrings directly in ``sqlitedict.py`` or call ``help(sqlitedict)``.
 
 **Beware**: because of Python semantics, ``sqlitedict`` cannot know when a mutable
-SqliteDict-backed entry was modified in RAM. For example, ``db["123"]["name"] = "hello world"``
-will leave ``mydict["123"]`` object as it was, not changing the name. You'll need to
+SqliteDict-backed entry was modified in RAM. You'll need to
 explicitly assign the mutated object back to SqliteDict:
 
 .. code-block:: python
 
-    val = db.get("123", {})
-    val["name"] = "hello world" # sqlite DB not updated here!
+    item = db["123"]
+    item["name"] = "hello world" # sqlite DB not updated here!
```

Comment:
Why should `db["123"]` be a dictionary? I find this change obfuscates more than it elucidates.

How about this:

```python
    dict_value = mydict.get("some_key", {})
    dict_value["inner_key"] = "hello world"  # sqlite DB not updated here!
```

Reply:
Because that's the interface from sqlitedict that acts like a dictionary? db is the sqlitedict class, db["key"] returns a row, the row is most likely a dict (I suppose it's possible to store a bare string or integer).

I find that the insistence to call the database "mydict" utterly confusing (I almost did not use the library in the first place because of that). The database object is not a dict, it acts like a dict in some aspects but it has nothing to do with a dict. Hence a major part of the documentation improvement is to call the database variable db instead of mydict.

Reply:
The question is about the `["123"]` part, not the `db` part.